src/main/java/nl/tudelft/cse1110/andy/writer/standard/StandardResultWriter.java

```diff
@@ -239,7 +240,7 @@ private void printModeAndTimeToRun(Context ctx, double timeInSeconds) {
     }
 
     private void printMetaTestResults(Context ctx, MetaTestsResult metaTests) {
-        if(!metaTests.wasExecuted())
+        if(!metaTests.wasExecuted() || metaTests.getTotalTests() == 0)
```

Comment:
This condition could be inside `MetaTestsResult`, e.g., `metaTests.noTestsExecuted()` or something like that

Reply:
Sure, I extracted it to `metaTests.hasNoMetaTests()` since this is the total number of all meta tests and not just the executed ones.